src/Bicep.Core.Samples/Variables_LF/main.diagnostics.bicep

```diff
@@ -122,3 +122,12 @@ var previousEmitLimit = [
     }
   }
 ]
+
+var myVar = 'hello'
+var myVar2 = any({
+  something: myVar
+})
+var myVar3 = any(any({
```

Comment:
Mind adding a test for `someFunction(any(...))` (e.g. any wrapped in a function), just to validate it generates correct expression syntax?

Reply:
Sure I can add those.